src/aiori-DFS.c

```diff
@@ -27,15 +27,15 @@
 #include <fcntl.h>
 #include <libgen.h>
 
+#include <mpi.h>
 #include <gurt/common.h>
 #include <gurt/hash.h>
 #include <daos.h>
 #include <daos_fs.h>
 
-#include "ior.h"
-#include "iordef.h"
 #include "aiori.h"
 #include "utilities.h"
+#include "iordef.h"
 
 dfs_t *dfs;
 static daos_handle_t poh, coh;
```

Comment:
I'm a bit concerned about the use of several global daos handles, instead of returning them as a pointer. 
Does this support a proper semantics opening multiple "files/objects" at a time?

Reply:
yes those are the pool & container handles.
The pool is the storage unit (controls capacity, storage reservation).
The container is the object address space. In POSIX world, think of it as a namespace.
so for IOR and mdtest, the expectation is always to have a single pool, and access to a single namespace (daos container / dfs mount).
A file in DFS is represented as an object (dfs_obj_t - see what is returned in DFS_Create). and from this model you can have billions/trillions of objects (files, directories) per namespace.

hope that is clear. for more info please see:
https://daos-stack.github.io/overview/storage/